project-reports/2024/2024-annual-Hyperledger-Besu.md

```diff
@@ -0,0 +1,45 @@
+
+# Maintainer Diversity
+
+[Maintainers File.](https://github.com/hyperledger/besu/blob/main/MAINTAINERS.md) Consensys Software makes up around 87% of the maintainer base, but is committed to the project. Maintainer diversity increased this year by adding Kaleido as a contributing organization, alongside contributions from Web3Labs. Swirlds Labs also has new contributors moving toward maintainer status in 2024. We are committed to increasing the diversity on this list by encouraging more development on the private network code-base by those using the technology. Existing maintainers have given special attention to new pull requests and are very timely in moving those forward. 
+
+
+# Project Adoption	
+
+The Besu project has been adopted by Linea, a layer 2 network developed by Consensys. It is now powering the sequencer that is used to produce blocks and advance the network as critical infrastructure. Citi Bank has also joined the Hyperledger Foundation as a member organization using Hyperledger Besu. Several new institutional staking providers have adopted Besu in their tech stacks to support Ethereum client diversity on Mainnet. Besu sits in 3rd place alongside four other execution clients. It is also being forked for use in other layer 2 networks. 
+
+There has been significant adoption around CBDC pilots, interested in building on performant technology and interoperable Ethereum/EVM standards. [This report](https://www.linuxfoundation.org/hubfs/Hyperledger_CBDC%20ebook_V2.pdf?hsLang=en) breaks down many in-flight CBDC projects building the Besu. 
+
+# Goals
+
+## Performance Against Prior Goals
+The Besu maintainers successfully delivered the Cancun hard fork on Ethereum Mainnet and are looking forward to scoping the Prague fork. The Besu project has struggled to court sets of new maintainers from organizations building both public and private networks, but we are hopeful the new financial services working group may yield new interest and contributions. We have made some progress on standardizing features across both network types, like shipping snap-sync as a server. This means that, shortly, private networks and Mainnet can use the same sync methodology (and we may remove fast-sync). We are still working on Bonsai Archive and other feature consolidation efforts. 
+
+The enterprise and public roadmaps were both delivered on the [wiki.](https://wiki.hyperledger.org/pages/viewpage.action?pageId=24781786). We are hopping this will help with adoption, clarity among developers and orgs using Besu, and will keep up the transparency efforts by Maintainers. 
```

Comment:
```suggestion
The enterprise and public roadmaps were both delivered on the [wiki.](https://wiki.hyperledger.org/pages/viewpage.action?pageId=24781786). We are hoping this will help with adoption, clarity among developers and orgs using Besu, and will keep up the transparency efforts by Maintainers. 
```